src/backend/joanie/debug/views.py

```diff
@@ -79,6 +82,81 @@ class DebugMailSuccessPaymentViewTxt(DebugMailSuccessPayment):
     template_name = "mail/text/order_validated.txt"
 
 
+class DebugMailSuccessInstallmentPaid(TemplateView):
+    """Debug View to check the layout of the success installment payment by email"""
+
+    def get_context_data(self, **kwargs):
+        """
+        Base method to prepare the document context to render in the email for the debug view.
+        Usage reminder :
+            /__debug__/mail/installment_paid_html
+        """
+        context = super().get_context_data()
+        product = ProductFactory(price=Decimal("1000.00"))
+        product.set_current_language("en-us")
+        product.title = "Test product"
+        product.set_current_language("fr-fr")
+        product.title = "Test produit"
+        product.save()
+        order = OrderGeneratorFactory(
+            product=product,
+            state=ORDER_STATE_PENDING_PAYMENT,
+            owner=UserFactory(first_name="John", last_name="Doe", language="en-us"),
+        )
+        invoice = Invoice.objects.create(
+            order=order,
+            parent=order.main_invoice,
+            total=0,
+            recipient_address=order.main_invoice.recipient_address,
+        )
+        for payment in order.payment_schedule[:2]:
```

Comment:
Since, we update `JOANIE_PAYMENT_SCHEDULE_LIMITS`, if the product price is less than 150, there will be only one installment, so this will raise an `IndexError`